### DISCUSSION TITLE
Non-networking devices

### DISCUSSION THREAD

**Discussion by aaronaxvig** [netbox-community/devicetype-library#2177] [Q&A]
I've got some racks with non-network rackmount equipment: paging (ceiling speaker) modules, amplifiers, AV switching, etc.  Given the lack of any such thing in the library so far, is it fair to conclude that it would not belong?

Is there any other library known to collect such misc. devices?

**Comment by jcralbino** [comment]
Indeed I believe it could be interesting also to have some templates for av devices

**Comment by ravench** [comment]
Generally, you can model anything that is a device and has connections in netbox. 
However, looking at the medium types, for example, there is coaxial cabling, but nothing for symetric/asymetric audio, for example, so it's probably not directly intended. 
However, there is also no cable type for analogue/ISDN phones, which is definitely something I'd expect to be in the scope of netbox. It might make sense to expand the data model in a reasonable way. (eg. by adding `2-wire` or similar a a cable type and the most important connectors)
Looking at Connectors, there are types for SMA, F and N already (Which are RF connectors) and of course RJ11. In my opinion, expanding those with `XLR`, `RCA` and `Screw Terminal` would not hurt either, but might be considered beyond scope by some.
